src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { DataService } from '../../services/DataService'
```

Comment:
I prefer to group globals on top and locals below and then alphabetical on each group

Reply:
It most common case to import react as first